api/tasks.go

```diff
@@ -366,9 +366,8 @@ func (m *MigrateStrategy) Copy() *MigrateStrategy {
 type VolumeRequest struct {
 	Name     string
 	Type     string
+	Source   string
 	ReadOnly bool `mapstructure:"read_only"`
```

Comment:
Do you expect any other config needed here for future proofing? Would those become fields directly here?

Reply:
We might add `config` back eventually (for ephemeral volumes) - but I can’t think of any future breaking changes we’d need to make here.